<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PamShow</class>
 <widget class="QDialog" name="PamShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_gray">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>总像素数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_res_1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>平均值：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_res_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>中位数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_res_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>标准差：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_res_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>最多灰度数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_res_5">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>曝光参数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_res_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>暗通道参数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_res_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
